### PR TITLE
Update Appendix B - Non Player Characters

### PR DESCRIPTION
**Pull request by MariosVeli**
Update of the stablocks and layout for Appendix B.

### CODE REVIEW

**Comment by orjanbp** [comment]
Just finished going through a pretty big slew of art updates for this doc. Took a while to make time, make the changes, edit the art, then holidays happened ... 😅

But this should be the most of it?

Some things still standing that are worth looking at: 

- Pg 1: The **acolyte** art here doesn't edit well, since the drawing crops on the hand and can't be easily extracted from the background. Maybe just remove it, extend the introductory text, and just have an inkblot going along the bottom? 

- Pg 2: Realising that the page-tearing here that keeps the **archmage** from bleeding onto the text could be adjusted a bit. Just around the Innate Spellcasting. 

- Pg 3: Under the **archdruid**, could maybe have an environmental art tear and a note? 

- Pg 5: Got some room for some sketch-art or a note next to the **berserker** here.

- Pg 6: Got it set in for making a paper note with the **crusader** here. Keep it or ditch it? There's room for one on the page just above or below, so three pages in a row with notes could be a bit much. 

- Pg 7: Feeling like having a paper note next to the **cultist's** head here. Could also just put in an inkblot that blends in with the paper tear, that could be cool. 

- Pg 10: Do something with the space below the **infiltrator** statblock? 

- Pg 12: Plenty of space for a paper note or some Scourge sketch art underneath the **necromancer** statblock.

- Pg 13: Kind of want to do something with the open space next to the **ranger** art. If Pg 12 had some sketchwork, and this page had a note, that could maybe work?

- Pg 16: Definitely gotta do something with the space to the right of the **tribal warrior** art here.

- Pg 17: A lot of space above the **warlock** art, as well as to the bottom left of the page. Doing much above the art might not be so necessary, but something more at the bottom could be nice? 

- Pg 18: A lot of space at the bottom, could just put an inkblot along it similar to how Pg 4 does it

- Changelog still need to be written. Suggest listing which of the NPC statblocks are from the SRD (20 of them) and which one are not (11 of them), which is all marked off in the big spreadsheet. 

Basically want to fill more empty space, have an even spread of notes written in. 🙂 Check off on how everything looks right now first, and then go from there? 

Here's a link to the doc that I've had it in while making edits: https://www.gmbinder.com/share/-MTLHvuaLKirneEgmUfF

**Comment by Tyloris** [comment]
I'm looking at the document with the edits. I already looked over the mechanical side of things with Geamros, so I'll stick to formatting.

- Page 1: The acolyte art looks fine to me.
- Page 2: The archmage looks fine.
- Page 3: The archdruid is kind of a mess. Accounting for all the subform abilities makes it take up the entire page, which kind of crowds out and hides the descriptor text in the bottom of the page. What I'd like to do is to have the descriptor text moved to the top and the statblock moved to the bottom of the page, with some art taking up the remaining space at the top. Another option is to add a flight form to the list of forms and then put in a variant block where the archdruid can prefer one form depending on which druidic order they belong to (Druids of the Claw, Fang, Talon, etc.)
- Page 5: I'd like to see one of the paper-note blocks here, that basically goes ""GRaaAaAAH!" -- An Average Berserker" (with the exact spelling mimicking the [bloodlust sound](https://www.youtube.com/watch?v=gOb5McbEl2M&ab_channel=smIndigo)).
- Page 6: Damnit, you've got a point about the paper notes. I really like this quote, too. That being said, the page is pretty full without it and the other pages need filling more, so it can probably go.
- Page 7: If we're cutting the paper note from crusader, then there should be one here. That being said, increasing the image size to 880px and the right offset to -45px fills up the page pretty well on its own. 
- Page 10: While you can fit a paper note here, I don't think it's necessary. If you do, a quote about SI:7 would work.
- Page 12: I agree with you on the need for art. I like [this](https://i.imgur.com/vyB3qN6.png) image, since paired with the necromancer you get the feeling that the skeleton was just raised at his command.
- Page 13: The page definitely needs something, though I'm not completely sure what. I did notice two non-art things on this page though: First, the priest is wearing armor again, even though I deliberately added the Inner Fire trait to make them more in line with the class. Should probably change the AC portion to state **13 (Inner Fire)**. Second, despite the fact that the ranger is the archer reworked to be closer to nature, it lacks any actual nature-based abilities. As an idea, first I'd give them Nature and Animal Handling as skills, and then have a variant/note block that covers what animal companions they might have with them; bear for dwarf, sabercat for night elf, etc. Pretty much just state a CR limit to the creature.
- Page 16: I think the troll art should be removed for this page. It doesn't really fit in with the others, which all have some sort of action and agency about them while this is just her standing idly. An option is one like [this](https://i.redd.it/1wh0y9jimzm21.jpg) that combines a troll and human fighting, therefore covering both page statblocks for the sake of art. My one complaint is that the human doesn't look weathered enough to be a veteran, though given Azeroth's unrealistic standard of beauty...
- Page 17: If you can find some sort of boulder/hill image, then you can put it along the bottom and imply that the warlock is looking down at an army she's about to smite into ruin.
- Page 18: An inkblot would be best.

**Comment by Tyloris** [comment]
Having trouble with putting in the changes myself, so I'm posting it here.

Page 12: Replace the image links with the following:
> 
> 
> 
> 
> 

---

Page 13: Add this note block to below the ranger description. Page numbers are left incomplete, to be revisited when the book is finalized. The artwork will need to be adjusted to work with the new layout; an example of a fix is listed below--depends on how much you want unobscured.

> > ##### Variant: Beast Companions
> > Rangers are often accompanied by beasts of the wild as partners in battle. This is usually a creature of <br/> CR 1 or below. Some common examples of tamed beasts are listed below.
> >
> > Creature		            | Preferred By  | Page  
> > --------------------------|---------------|----:
> > Brown Bear                | Dwarves       | ###
> > Dire Wolf                 | Humans, Orcs  | ###
> > Giant Moth                | Draenei       | ###
> > Raptor                    | Trolls        | ###
> > Sabercat                  | Night elves   | ###
> > Turtle                    | Pandaren      | ###
> > Wind Serpent              | Tauren        | ###
> > Young Dragonhawk          | Blood elves   | ###

> 
> 

---

Page 16: Add a 480px top margin above each statblock, then replace the art links with the following:

> 
>
